mint.json

```diff
@@ -127,118 +126,139 @@
     },
     {
       "group": "Architecture",
-      "pages": ["architecture/introduction", "architecture/diagrams"]
-    },
-    {
-      "group": "Build Workflows",
       "pages": [
-        "workflow/introduction",
-        "workflow/content",
-        "workflow/digest",
-        "workflow/delay"
+        "architecture/overview",
+        "architecture/diagrams"
       ]
     },
-         
     {
-      "group": "Channels",
```

Comment:
@iampearceman I don't really like this 4 level nesting here :( Let's either flatten it, so each channel is in it's own row and not inside the Step Channels menu, or move it to it's own section like it was before

Reply:
Resolved.